### DISCUSSION TITLE
Which theme do we look at in the presentation video ?

### DISCUSSION THREAD

**Discussion by francklays** [kovidgoyal/kitty#5990] [Q&A] [answered]
Is there a way to get the theme or colors seen in kitty in the presentation video here https://sw.kovidgoyal.net/kitty/ ?

**Comment by kovidgoyal** [comment, marked as answer]
I'm afraid I dont recall what theme I used back then, however, I
currently use Dimmed Monokai.

On Tue, Feb 07, 2023 at 10:20:05AM -0800, Franck Laÿs wrote:
> Is there a way to get the theme or colors seen in kitty in the presentation video here https://sw.kovidgoyal.net/kitty/ ?
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/5990
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Comment by francklays** [comment]
Thank you Kovid
